### PR TITLE
update!: Post three job openings

### PR DESCRIPTION
**Pull request by jonaharagon**

### CODE REVIEW

**Review by kimg45 — APPROVED** [review]

File: docs/about/jobs/journalist.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Your responsibilities will include, but aren’t limited to:
+
+- Creating high-quality articles for our [knowledge base](../../basics/why-privacy-matters.md).
+- Performing product reviews for our [reviews](https://www.privacyguides.org/articles/category/reviews/) section and [tool recommendations](../../tools.md).
+- Researching new topics to cover.
+- Interviewing and fact-checking all relevant sources.
+- Regular posting of high-quality, unbiased journalistic content across our platforms.
+
+As a guideline, we expect roughly 3-5 articles a week that meet our quality standards, since we believe a well-researched article will take at least 8 hours to research and write on average. We realize some articles can be completed quickly, while others may take weeks or longer before publishing. You will be empowered to use your best judgement and prioritize your work accordingly.
```
Comment (freddy-m) [inline docs/about/jobs/journalist.md:24]: Do we want quality or quantity? Of course ideally we want both, but posting two great articles a week is better to me than churning out five mediocre ones.

**Review by blacklight447 — APPROVED** [review]

File: docs/about/jobs/journalist.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+We are much more interested in articles that deeply cover the subject area than articles that cover the news of the day.
+
+Job requirements:
+
+- Excellent organization and communication skills.
+- Expertise in English and writing.
+- Flexibility to set and respond to varying priorities and deadlines.
+- Proactive, results-driven mindset with a strong sense of initiative.
+- Personal interest in consumer privacy, cybersecurity, and technology.
+- Regularly communicating with the Privacy Guides committee and other team members.
```
Comment (redoomed1) [inline docs/about/jobs/journalist.md:39]: This line can be moved to the list of responsibilities (as it is in the Content Creator posting) or
```suggestion
- Regular communication with the Privacy Guides committee and other team members.
```
for consistency.

**Review by redoomed1 — COMMENTED** [review]
Some suggestions to make the job postings more consistent in terms of the way info is organized, and to remove unnecessary spaces so as to appease MegaLinter

File: docs/about/jobs/content-creator.md [hunk 3]
```diff
@@ -5,18 +5,24 @@ description: Privacy Guides is looking for a video producer and host for informa
 
 [:material-arrow-left-drop-circle: Job Openings](../jobs.md)
 
-If you have ever dreamed of being a YouTuber and combating misinformation about privacy and cybersecurity on the internet, this is the position for you!
+Are you passionate about privacy and cybersecurity?
```
Comment (jonaharagon) [inline docs/about/jobs/content-creator.md:8]: Why this change?

File: docs/about/jobs/journalist.md (hunk 1 above)
Comment (jonaharagon) [reply docs/about/jobs/journalist.md:28]: Line 18 above says:

> Our primary mission is to publish the highest quality content surrounding consumer privacy and cybersecurity on the internet, not the highest quantity of stories.

**Review by kimg45 — APPROVED** [review]

**Review by blacklight447 — APPROVED** [review]

File: docs/about/jobs/journalist.md (hunk 1 above)
Comment (freddy-m) [reply docs/about/jobs/journalist.md:28]: I suppose it does say roughly as well

**Review by kimg45 — APPROVED** [review]
